.eslintrc.json

```diff
@@ -1,11 +1,38 @@
 {
-  "extends": ["react-app", "plugin:@typescript-eslint/recommended"],
+  "extends": ["react-app", "plugin:@typescript-eslint/recommended", "prettier"],
   "overrides": [
     {
       "files": ["**/*.tsx", "**/*.ts"],
       "rules": {
-        "@typescript-eslint/explicit-function-return-type": "off"
+        "@typescript-eslint/brace-style": ["error", "1tbs", { "allowSingleLine": true }],
+        "@typescript-eslint/explicit-function-return-type": "off",
+        "@typescript-eslint/func-call-spacing": ["error"],
+        "@typescript-eslint/member-ordering": ["warn"],
+        "@typescript-eslint/no-require-imports": ["error"],
+        "react/no-unused-prop-types": ["warn", { "skipShapeProps": true }],
+        "array-bracket-spacing": ["warn", "never"],
+        "import/order": [
+          "error",
+          {
+            "groups": [
+              "builtin",
+              "external",
+              ["internal", "parent", "sibling", "index"]
+            ],
+            "newlines-between": "always"
+          }
+        ],
+        "max-len": ["warn", { "code": 120 }],
+        "no-console": "warn",
+        "no-plusplus": "error",
+        "no-undef": "warn",
+        "object-curly-spacing": ["warn", "always"],
+        "sort-keys": ["warn", "asc", { "caseSensitive": true, "natural": true, "minKeys": 2 }]
       }
     }
-  ]
+  ],
+  "plugins": ["prettier"],
+  "rules": {
+    "prettier/prettier": "error"
+  }
```

Comment:
This can be merged with the `rules` field above

Reply:
We remove this by using recommended settings